pandas/core/series.py

```diff
@@ -1462,63 +1463,21 @@ def dot(self, other):
         else:  # pragma: no cover
             raise TypeError('unsupported type: %s' % type(other))
 
-    def searchsorted(self, v, side='left', sorter=None):
-        """Find indices where elements should be inserted to maintain order.
+    @Substitution(klass='Series')
+    @Appender(base._shared_docs['searchsorted'])
+    def searchsorted(self, v, side='left', *args):
+        validate_args(args, min_length=0, max_length=1)
+        sorter = None if not args else args[0]
 
-        Find the indices into a sorted Series `self` such that, if the
-        corresponding elements in `v` were inserted before the indices, the
-        order of `self` would be preserved.
+        if not isinstance(self._values, Index):
+            if sorter is not None:
+                sorter = com._ensure_platform_int(sorter)
 
```

Comment:
@jreback, @shoyer: Is there any reason why `Series` was the only class that really supported `sorter`?  I know that its `_values` attribute is a `numpy` array, but in the spirit of the other classes, does it make sense just to remove this?  It seems somewhat inconsistent to try to maintain this but drop support of `sorter` everywhere else.